tools/cmake/cores/Cortex-A9.cmake

```diff
@@ -9,6 +9,7 @@ if(${MBED_TOOLCHAIN} STREQUAL "GCC_ARM")
         "-mfpu=vfpv3"
         "-mfloat-abi=hard"
         "-mno-unaligned-access"
+        ""-mcpu=cortex-a9""
```

Comment:
Excessive quoting. Shouldn't this replace the `-march=armv7-a`?

Reply:
Good point, checked https://community.arm.com/developer/tools-software/tools/b/tools-software-ides-blog/posts/compiler-flags-across-architectures-march-mtune-and-mcpu 

`On Arm, -mcpu combines -march and -mtune.4 On x86, -mcpu and -mtune are the same.5` - I'll remove it .

Reply:
This is for Gcc: 
`This specifies the name of the target ARM processor. GCC uses this name to derive the name of the target ARM architecture`